settings.gradle.kts

```diff
@@ -7,6 +9,9 @@ plugins {
 dependencyResolutionManagement {
     repositories {
         mavenCentral()
+        maven {
+            url = URI.create("https://oss.sonatype.org/content/repositories/snapshots/")
+        }
```

Comment:
I think we should be able to get this merged once the detekt team releases a RC in maven central (and this is not necessary)

Reply:
sounds good. will revert this when we get a proper pre-release, just wanted CI to pass in mean time